tests/test_security.py

```diff
@@ -91,6 +91,31 @@ def test_template_data(self, env):
         )
         assert escape(t.module.say_hello("<blink>foo</blink>")) == escaped_out
 
+    def test_template_data_custom_escape(self, env_custom_autoescape):
+        env = env_custom_autoescape
+        t = env.from_string(
+            "{% macro say_hello(name) %}"
+            "$<p>Hello {{ name }}!</p>${% endmacro %}"
+            '{{ say_hello("<$$foo$$>") }}'
+        )
+        my_escape = env.default_markup_class.escape
+        escaped_out = "$<p>Hello <€€foo€€>!</p>$"
+        assert t.render() == escaped_out
+        # Make sure that the same Markup class is used everywhere
+        # somehow even different instances of the escape class are use
```

Comment:
```suggestion
        # somehow even different instances of the escape class are used
```

Reply:
Actually I thought you would only correct the documentation and not the comments as well.
But I guess the point about comments is that people understand them, so it makes sense. But well yeah so MANY misstakes... sorry :(